app/Http/Middleware/SimulateError.php

```diff
@@ -24,10 +25,12 @@ class SimulateError
      */
     public function handle(Request $request, Closure $next)
     {
-        if ($request->header('X-Mismatch-Finder-Error') == $request->path()) {
-            throw new RuntimeException("Simulated Server Error");
-        } elseif ($request->header('X-Mismatch-Finder-Not-Found') == $request->path()) {
-            abort(404);
+        if (!app()->environment('production')) {
+            if ($request->header('X-Mismatch-Finder-Error') == $request->path()) {
+                throw new RuntimeException("Simulated Server Error");
+            } elseif ($request->header('X-Mismatch-Finder-Not-Found') == $request->path()) {
+                abort(404);
+            }
```

Comment:
Nice! A small style nitpick, in this case, I'd personally prefer to use a guard rather than wrap conditionals with conditionals. So the suggestion would be as follows:

```suggestion
        if (app()->environment('production')) {
            return;
        }
        
        if ($request->header('X-Mismatch-Finder-Error') == $request->path()) {
            throw new RuntimeException("Simulated Server Error");
        } elseif ($request->header('X-Mismatch-Finder-Not-Found') == $request->path()) {
            abort(404);
```